webui/react/src/e2e/tests/userManagement.spec.ts

```diff
@@ -85,51 +35,196 @@ test.describe('User Management', () => {
         userManagementPageSetupTeardown = new UserManagement(pageSetupTeardown);
         await authFixtureSetupTeardown.login();
       });
-
-      await test.step('Create User', async () => {
-        await userManagementPageSetupTeardown.goto();
-        testUser = await userFixtureSetupTeardown.createUser();
-      });
     });
 
     test.afterAll(async () => {
       await userManagementPageSetupTeardown.goto();
       await test.step('Deactivate User', async () => {
-        // BUG [INFENG-628] Users page loads slow
-        await expect(async () => {
-          await userFixtureSetupTeardown.deactivateTestUsers();
-        }).toPass({ timeout: 20_000 });
+        await userFixtureSetupTeardown.deactivateAllTestUsers();
```

Comment:
Doesn't this cause problems for other describe blocks if they make a user during the test?

Reply:
doesn't user fixture pull the users from the table though with this? `const ids = await this.userManagementPage.table.table.allRowKeys();` and then on line #145 we set the user to the map.

Reply:
okay so two things. the part you mentioned is `deactivateTestUsersOnTable`. `table.allRowKeys();` in that function is used to manage the internal state. "physically", the test is using this to set the status in the product.
```
    await this.userManagementPage.table.table.headRow.selectAll.pwLocator.click();
```

however, this line here is referencing `deactivateAllTestUsers`. That method will only go through the users from this test session. It manually searches for each user one at a time, so there shouldn't be a risk of search collision.
```
    for await (const user of users.values()) {
```